docs/installing-northstar/using-northstar/playing-on-linux.md

```diff
@@ -84,3 +83,33 @@ Some users have reported issues with enabling LatencyFleX. If you see `"Unable t
 ### Game crashes on launch with Cause: Access Violation Data Execution Prevention (DEP) at: 0x00000000
 
 try running with [ProtonGE](https://github.com/GloriousEggroll/proton-ge-custom/).
+
+### Reducing stuttering (Wine and Proton)
+>
+_Pre-compiled DXVK cache_: https://github.com/begin-theadventure/dxvk-caches/raw/main/Titanfall%202/Titanfall2.dxvk-cache.tar.xz
+>
+Proton: extract and put in path/steamapps/shadercache/1237970/DXVK_state_cache
+>
+Wine: extract and put it next to game's .exe. Also remember to rename it if the .exe has a different name)
+>
+_DXVK-async_:
+>
+Wine: https://github.com/Sporif/dxvk-async/releases (put it in ~/.local/share/lutris/runtime/dxvk extract and type name in "DXVK version" of the folder, to enable set the environment variable _DXVK_ASYNC 1_)
+>
+Proton: can be used with **Proton-GE**. Type _DXVK_ASYNC 1_ under LAUNCH OPTIONS.
+>
+_Prevent Origin from writing certain files_: https://github.com/ValveSoftware/Proton/issues/4001#issuecomment-647014231
+>
+Path:
+>
+Proton: x/steamapps/compatdata/1237970/pfx/drive_c/users/steamuser/Application Data/Origin
+>
+Wine: x/drive_c/users/yourusername/AppData/Roaming/Origin
+>
+Access can be restricted using a file manager or terminal:
+>
+_chmod -R 555_  ->access only
+> 
+_chmod -R 755_  ->access + save
+>
+It's also possible to make aliases to type something short like tfoff/tfon.
```

Comment:
I'd still keep the explanation of the cause of the stutter. The updated description makes it seem like this is a necessary step to get the game playable which it certainly is not. So maybe also mention that using pre-compiled shaders is simply an optimisation steps for those not wanting to wait for shader compilation.

Also, what's with all the `>`? ^^

Reply:
Would "If you feel the game is stuttering you might want to try this list of optimizations." be enough?

Reply:
> Would "If you feel the game is stuttering you might want to try this list of optimizations." be enough?

https://github.com/R2Northstar/NorthstarWiki/pull/58/commits/3babd237ab343f2f12e73bfd1f90ccc00b81749b

Reply:
additionally https://github.com/R2Northstar/NorthstarWiki/pull/58/commits/224711d3f392e767032606a6f2c11731fa422d1e

Reply:
I still wonder how to.
Just pasting it kinda doesn't make sense as there's also section about Origin which isn't cache, similar is with dxvk-async.

Reply:
Check https://github.com/R2Northstar/NorthstarWiki/commit/2fe101285511b6a0660aefbc9a842b4888f3ddb0, please

Reply:
https://github.com/R2Northstar/NorthstarWiki/pull/58/commits/a1b914cbfd8298dd2307efbd00d59e845db60f97